eradiate/scenes/atmosphere/_homogeneous.py

```diff
@@ -16,11 +24,42 @@ class HomogeneousAtmosphere(Atmosphere):
     """
     Homogeneous atmosphere scene element [:factorykey:`homogeneous`].
 
-    This class builds an atmosphere consisting of a homogeneous medium.
-    Scattering uses the Rayleigh phase function.
+    This class builds an atmosphere consisting of a homogeneous medium with
+    customisable collision coefficients and phase function, attached to a
+    cuboid shape.
     """
 
-    sigma_s = documented(
+    _bottom: pint.Quantity = documented(
+        pinttr.ib(
+            default=ureg.Quantity(0.0, ureg.km),
+            units=ucc.deferred("length"),
+            converter=pinttr.converters.to_units(ucc.deferred("length")),
+            validator=pinttr.validators.has_compatible_units,
+        ),
+        doc="Atmosphere's bottom altitude.\n\nUnit-enabled field (default: ucc[length])",
+        type="float",
+        default="0 km",
+    )
+
+    _top: pint.Quantity = documented(
+        pinttr.ib(
+            default=ureg.Quantity(10.0, ureg.km),
+            units=ucc.deferred("length"),
+            converter=pinttr.converters.to_units(ucc.deferred("length")),
+            validator=pinttr.validators.has_compatible_units,
+        ),
+        doc="Atmosphere's top altitude.\n\nUnit-enabled field (default: ucc[length]).",
+        type="float",
+        default="10 km.",
```

Comment:
Any reason to change the default from 100 to 10 km?

Reply:
100 km felt a bit too large, given that the atmosphere is homogeneous. 
The air number density' scale height is roughly 10 km so that when one approximates the earth's atmosphere as a homogeneous medium, it is somehow reasonable to me to assume that there is air below 10 km and nothing above.
Of course, the thermophysical variables vary significantly in that 10 km "homogeneous" atmosphere so the assumption is very crude, but 100 km would be even worse.
Also, 10 km is approximately the height of the troposphere, the first layer of the Earth's atmosphere.
Altogether, it made some sense to go for that value for a homogeneous atmosphere.